### PR TITLE
OCLOMRS-54: A user should be able to view their personal dictionaries and dictionaries belonging to all other organizations and users

### PR DESCRIPTION
**Pull request by mwaz**
# JIRA TICKET NAME:
[OCLOMRS-54](https://issues.openmrs.org/browse/OCLOMRS-54)

# Summary:
As a user after adding dictionaries, I want to view my dictionaries and also the dictionaries for all other OpenMRS organizations and users. 
A filter should be put in place to enable a user toggle between their dictionaries and all the dictionaries on the platform, the default should, however, be the dictionaries that have been created by the logged in user, either as personal dictionaries or as dictionaries belonging to their organizations.

### CODE REVIEW

File: src/components/dashboard/styles/index.css [hunk 1]
```diff
@@ -381,3 +381,12 @@ nav {
...
 }
+.dashboard-alert{
+  background: #e9e9e9;
+  color: #000000!important;
+  text-align: center;
+}
+.search-filter {
+  padding-right: 5%;
+  float: right;
+}
```
Comment (shakirandagire) [inline src/components/dashboard/styles/index.css]: Could you add a line at the end of the file

Comment (hadijahkyampeire) [inline src/components/dashboard/styles/index.css]: new line needed here

Comment (mwaz) [reply src/components/dashboard/styles/index.css]: resolved

Comment (mwaz) [reply src/components/dashboard/styles/index.css]: resolved

**Comment by dkayiwa** [comment]
Can you look into these merge conflicts?

**Comment by mwaz** [comment]
@dkayiwa resolved!

**Pull request merged**